### ISSUE TITLE
Replace empty tab instead of opening new tab

### ISSUE THREAD

**Issue by redstreet** [ctrlpvim/ctrlp.vim#430]
If the current buffer is empty and unmodified (i.e., if I just started up vim by typing 'vim' and did nothing else), then when I bring up CtrlP and hit <C-t>, I'd like it to replace my current buffer and current tab rather than opening a new buffer in a new tab.

Is there a way to accomplish this via settings? One obvious way to do this is to press <enter> instead, but on subsequent selects press <C-t>. But I'd like to use muscle memory and not type something different based on context.

TL; DR: Can the very first open in the tab mode replace the current tab?

Thank you.

**Comment by mattn** [comment]
you can do it with g:ctrlp_open_func.
